src/algebra/group/hom.lean

```diff
@@ -235,6 +237,22 @@ instance {M G} [monoid M] [comm_group G] : comm_group (M →* G) :=
 
 end monoid_hom
 
+namespace add_monoid_hom
+
 /-- Additive group homomorphisms preserve subtraction. -/
-@[simp] theorem add_monoid_hom.map_sub {G H} [add_group G] [add_group H] (f : G →+ H) (g h : G) :
+@[simp] theorem map_sub {G H} [add_group G] [add_group H] (f : G →+ H) (g h : G) :
   f (g - h) = (f g) - (f h) := f.map_add_neg g h
+
+/-- Left multiplication by an element of a (semi)ring is an `add_monoid_hom` -/
+def left_mul {R : Type*} [semiring R] (r : R) : R →+ R :=
```

Comment:
Browsing through ring_theory.algebra I realise that this should perhaps be called mul_left, and similarly mul_right below.

Reply:
Yep, that seems right.